packages/sit-onyx/src/types/fonts.ts

```diff
@@ -1,2 +1,7 @@
 export const TEXT_SIZES = ["small", "default", "large"] as const;
 export type TextSize = (typeof TEXT_SIZES)[number];
+
+// when adding a new truncation type, make sure to also add CSS for it
+// inside styles/fonts.scss
```

Comment:
```suggestion
/** when adding a new truncation type, make sure to also add CSS for it
 * inside styles/fonts.scss
 */
```
or intentional not in jsdocs style? :)

Reply:
Intentional because otherwise it would appear when projects import it which can be confusing to them